app/controllers/api/v1/registrant/companies_controller.rb

```diff
@@ -0,0 +1,52 @@
+require 'serializers/registrant_api/company'
+
+module Api
+  module V1
+    module Registrant
+      class CompaniesController < ::Api::V1::Registrant::BaseController
+        MAX_LIMIT = 200
+        MIN_OFFSET = 0
+
+        def index
+          result = error_result('limit') if limit > MAX_LIMIT || limit < 1
+          result = error_result('offset') if offset < MIN_OFFSET
+          result ||= companies_result(limit, offset)
+
+          render result
+        end
+
+        def current_user_companies
+          current_registrant_user.companies
+        rescue CompanyRegister::NotAvailableError
+          nil
```

Comment:
Shouldn't it be an empty array? `companies_result` would fail if it was nil I think.

Reply:
Yep, you are right, thanks.